corehq/apps/users/login_record.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+from django_redis import get_redis_connection
+
+ONE_DAY = 60 * 60 * 24
+EPOCH = datetime.utcfromtimestamp(0)
+
+
+class LoginRecord:
```

Comment:
Does this track all logins or just login failures? My read of it is just failed attempts, but I could be missing something.

Depending on the answer, what do you think about changing the name. Maybe something like `RecentLoginAttemptInfo` or `RecentFailedLoginAttemptsCounter` or something like that. `LoginRecord` to me sounds like a permanent log row ("record") of every login ever made (which is something we've talked about but not what this class is doing, so it was a little confusing to me).